agr_literature_service/api/tests/test_001_reference.py

```diff
@@ -180,14 +183,42 @@ def test_reference_large():
                 ]
             }
         ],
+        "ontologies": [
+            {
+                "ontology_id": "Ontology1",
+                "mod_abbreviation": "RGD_ont",
```

Comment:
Oh, does posting this full_xml create a MOD entry with that mod_abbreviation ?  I would have thought it would fail because it doesn't exist.  Or do the RGD_ont + FB_ont mods get created sometime before the first test runs ?